cdap-hbase-compat-base/src/main/java/co/cask/cdap/data2/util/hbase/HBaseTableUtil.java

```diff
@@ -498,6 +498,12 @@ public void truncateTable(HBaseDDLExecutor ddlExecutor, TableId tableId) throws
     ddlExecutor.truncateTable(tableName.getNamespaceAsString(), tableName.getQualifierAsString());
   }
 
+  public void grantPrivileges(HBaseDDLExecutor ddlExecutor, TableId tableId,
+                              Map<String, String> privileges) throws IOException {
```

Comment:
Then why do we have it truncate()? I guess it still encapsulates the correct logic to map the table name to an HBase table name.